app/models/event.js

```diff
@@ -0,0 +1,34 @@
+var Sequelize = require('sequelize');
+
+var schema = {
+	payload: {
+		type: Sequelize.JSONB,
+		allowNull: false
+	},
+	type: {
+		type: Sequelize.STRING,
+		allowNull: false
+	},
+	dispatchedAt: {
+		type: Sequelize.DATE,
+		allowNull: false,
+		field: 'dispatched_at'
+	},
+};
```

Comment:
Related -- we may want to store the source of the event too, unless we're only interested in storing outbound events